rules/linux/lnx_system_info_discovery.yml

```diff
@@ -0,0 +1,47 @@
+action: global
+title: System Information Discovery
+id: 42df45e7-e6e9-43b5-8f26-bec5b39cc239
+status: stable
+description: Detects system information discovery commands
+author: Ömer Günal, oscd.community
+date: 2020/10/08
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1082/T1082.md
+falsepositives:
+    - Legitimate administration activities
+level: low
+tags:
+    - attack.discovery
+    - attack.t1082
+---
+logsource:
+    product: linux
+    categories: process_creation
+detection:
+    selection:
+        CommandLine|contains:
+          - 'uname'
+          - '/proc/version'
+          - '/etc/*-release'
+          - 'hostname'
+          - '/etc/issue'
+          - 'uptime'
+          - 'lspci'
+          - 'dmidecode'
+          - 'lscpu'
+          - 'lsmod'
```

Comment:
Could you please split processname and commandlines here? it will improve performance dramatically. 
also, as you are using `contains` here, you need to get rid of the asterisk here — `/etc/*-release`, as it is going to be considered as part of the search, not as a mask, 
Thank you!